tools/mason/MasonHelp.chpl

```diff
@@ -344,15 +344,19 @@ proc masonCompilerHelp() {
  }
 
 proc masonTestHelp() {
-  writeln("Run test files located within target/debug/test");
+  writeln("mason test works inside and outside of mason packages.");
+  writeln("Inside a mason package run test files located within target/debug/test");
+  writeln("Outside of a mason package runs all the files in the provided path or the current directory.");
```

Comment:
```suggestion
  writeln("Outside of a mason package: run test files found in the provided path (defaults to '.').");
```

Reply:
Thanks for the feedback. I have made the changes.